### DISCUSSION TITLE
What do you gain by being a member of the Athens Community?

### DISCUSSION THREAD

**Discussion by avidrucker** [athensresearch/athens#1223] [General]
@seekanddefine and I are looking to better understand our community and would like to know why you are contributing to Athens and if there are things that you might like to experience within our community here on out as well. Please upvote and/or add an emoji to anything below that resonates with you, or add new items as you think of them 🙂👍

**Comment by avidrucker** [comment]
Diversify my coding language knowledge

**Comment by avidrucker** [comment]
Gain experience in software development

**Comment by avidrucker** [comment]
Develop my portfolio

**Reply by avidrucker** [reply]
Example question: "I want to showcase my contributions to Athens on my own GitHub page, how do I do that?"

**Comment by avidrucker** [comment]
Learn new skills

**Comment by avidrucker** [comment]
Make new friends

**Comment by avidrucker** [comment]
Network professionally

**Comment by avidrucker** [comment]
Make a contribution to Open Source

**Comment by avidrucker** [comment]
Engage in a creative endeavor with others

**Comment by avidrucker** [comment]
Expand the idea of what open source communities can do

**Comment by avidrucker** [comment]
Engage with others to address community issues

**Comment by avidrucker** [comment]
Make the world a better place

**Comment by avidrucker** [comment]
Change the status quo / disrupt systems

**Comment by avidrucker** [comment]
Envision and build a better future

**Comment by avidrucker** [comment]
Cultivate new audiences

**Comment by datokrat** [comment]
Find an efficient and enjoyable way to handle the abundant amount of knowledge in the world and enrich each other's world views
